pkgs/top-level/aliases.nix

```diff
@@ -1670,6 +1670,7 @@ mapAliases {
   waypoint = throw "waypoint has been removed from nixpkgs as the upstream project was archived"; # Added 2024-04-24
   wcm = throw "'wcm' has been renamed to/replaced by 'wayfirePlugins.wcm'"; # Add 2023-07-29
   webkitgtk_5_0 = throw "'webkitgtk_5_0' has been superseded by 'webkitgtk_6_0'"; # Added 2023-02-25
+  webkitgtk = lib.warn "Explicitly set the ABI version of 'webkitgtk'" webkitgtk_4_0;
```

Comment:
Moving it to aliases without getting rid of them from  tree will break eval with `!allowAliases` on the commit. This might be annoying e.g. when bisecting. Either keep the `webkitgtk = webkitgtk_4_0` in `all-packages.nix` in the first commit and change it to aliases in the second one, or squash the commits.